lib/install/config/webpacker.yml

```diff
@@ -34,6 +34,7 @@ development:
     host: localhost
     port: 8080
     https: false
+    hot_reloading: false
```

Comment:
I'm fine with HMR as well, though that raises a larger question. Now that we have lazy compilation available without a separate dev server, isn't HMR the _only_ reason why you'd use the dev server? If so, maybe it should be called the webpack-hmr-server and focus on that task exclusively.